### PR TITLE
disable current documentation build script.

### PR DESCRIPTION
**Pull request by abuts**
Fixes Re #6.  This is simplest solution to failure. No changes to documentation are done by the modified script as the code which does this is commented. Now, when the documentation is build by github actions this is probably reasonable solution.

GithubActions which build release documentation should be fixed by another ticket but this is minor issue as current documentaiton is updated on merge to master anyway.

### CODE REVIEW

File: deploy.Jenkinsfile
Comment (cmarooney-stfc) [inline deploy.Jenkinsfile:1]: (a) it is totally unclear how the changes do the disable. Presuming the removal of the echo line does the disable, but really the description in the PR needs to say that. We cannot rely on everyone knowing what echo does in PS
(b) the PR description has no reference to a ticket and no description of why the script is disabled. These things have to be spelled out so people can follow the process. Why is this being done?

**Review by cmarooney-stfc — CHANGES_REQUESTED** [review]
See file comment in the PR

File: deploy.Jenkinsfile
Comment (abuts) [reply deploy.Jenkinsfile:1]: Unfortunately, I dealt with this issue due to inability to make the release, so I need to test the changes and I do not have aceess to Jenkins to run test scripts. As the result, I need to test my changes by merging them first.  You will see them if open first commit as second have to resolve conflict, caused by doing the same thing twice.

The disable is already in master, the lines responsible for pushing the doc are commented and the script works:
https://anvil.softeng-support.ac.uk/jenkins/job/PACE-neutrons/job/Horace/job/Deploy/

Here I am just removing debug strings, I put to understand where the failure comes from.

Comment (abuts) [reply deploy.Jenkinsfile:1]: There are no much issues in this repository, so the only one mentioned is sorted by the PR. 
I've added reference to it for your convenience.

**Pull request merged**